collectors/rgw.go

```diff
@@ -0,0 +1,188 @@
+package collectors
+
+import (
+	"encoding/json"
+	"log"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const RGWGCTimeFormat = "2006-01-02 15:04:05"
+const RAGOSGWAdminPath = "/usr/bin/radosgw-admin"
+
+type rgwTaskGC struct {
+	Tag     string `json:"tag"`
+	Time    string `json:"time"`
+	Objects []struct {
+		Pool     string `json:"pool"`
+		OID      string `json:"oid"`
+		Key      string `json:"ky"`
+		Instance string `json:"instance"`
+	} `json:"objs"`
+}
+
+// Expires returns the timestamp that this task will expire and become active
+func (gc rgwTaskGC) Expires() time.Time {
```

Comment:
minor: `ExpiresAt()` might be a little more apt because this returns a timestamp?